template/tailwind.config.js

```diff
@@ -0,0 +1,24 @@
+const merge = require('lodash.merge');
+const plugin = require('tailwindcss/plugin');
+const config = require('@studiometa/tailwind-config');
+
+/**
+ * TailwindCSS Configuration File
+ *
+ * @docs    https://tailwindcss.com/docs/configuration
+ * @default https://github.com/studiometa/tailwind-config/blob/develop/src/index.js
+ */
+module.exports = merge(config, {
+  // Extends the default Studio Meta Tailwind configuration here...
+  plugins: [],
+  theme: {},
+  purge: {
+    // Learn more on https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
+    enabled: process.env.NODE_ENV === 'production',
```

Comment:
This is the default value, we do not need to specify it.

Reply:
I'm not sure to understand your feedback, did you mean that also for line 17 ?

Reply:
From the doc:

> Now whenever you compile your CSS with NODE_ENV set to production, Tailwind will automatically purge unused styles from your CSS.

The default behaviour of Tailwind CSS with Purge CSS is to enable it only when `NODE_ENV` is set to `production`. So the configuration of `enabled` (line 17) is redundant with the default one and therefore does not need to be specified.